workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/DecorativeViewFactory.kt

```diff
@@ -5,8 +5,111 @@ import android.view.View
 import android.view.ViewGroup
 import kotlin.reflect.KClass
 
+/**
```

Comment:
This is still deprecated, but its seems obnoxious to delete its copious kdoc when there is no direct analog in the new world.